internal/security/gnxi/rpcexec.go

```diff
@@ -505,10 +513,17 @@ func GnsiAuthzRotate(ctx context.Context, dut *ondatra.DUTDevice, opts []grpc.Di
 	if err != nil {
 		return err
 	}
-	_, err = gnsiCStream.Recv()
-	// invalid policy is expected since the empty policy is not allowed
-	if strings.Contains(err.Error(), "invalid policy") || status.Code(err) == codes.InvalidArgument {
-		return nil
+	for {
+		_, err := gnsiCStream.Recv()
+		if err != nil {
```

Comment:
could this be a swith err {} instead of this chain or else/if/etc?

Reply:
Replace if else section with switch case in new commit